scripts/build-shaders.js

```diff
@@ -33,12 +33,15 @@ var data = data.split('\n').map(function(line) {
         } else {
             return params[1] + params[3] + 'p ' + params[4] + ' ' + params[5] + ' = u_' + params[5] + ';';
         }
+    } else if (line.match(pixel_ratio_regex)){
+        return '\n#define DEVICE_PIXEL_RATIO 2.0\n' + line;
```

Comment:
It actually inserts the hardcoded definition before (`#define DEVICE_PIXEL_RATIO 2.0`) before the matching line in the line-*.vertex shaders. so the output is

```glsl
#define DEVICE_PIXEL_RATIO 2.0
#define ANTIALIASING 1.0 / DEVICE_PIXEL_RATIO / 2.0
```

I've confirmed the build-shaders script is producing the `.hpp` shaders with the expected result but at least one of the shaders is not compiling and I can't find a helpful error message ☹️